src/renderer/components/main/mainbar/statusbar.tsx

```diff
@@ -0,0 +1,33 @@
+
+/* IMPORT */
+
+import * as React from 'react';
+import {connect} from 'overstated';
+import Main from '@renderer/containers/main';
+import Position from './statusbar_position';
+import WordCount from './statusbar_word_count';
+
+/* STATUSBAR */
+
+const Statusbar = ({ isStatusbar }) => {
+  if (!isStatusbar) {
+    return null;
+  }
+
+  return <div id="mainbar-statusbar" className="layout-header centerer">
```

Comment:
This should be `layout-footer`, and since you're using the spacer there shouldn't be a need for `centerer`

Reply:
The default stylings provided by layout-footer look... pretty bad, for the statusbar to be frank. There'd need to be some changes to the CSS for it to look congruent.

With "layout-header centerer"
![image](https://user-images.githubusercontent.com/1127397/50730341-31457180-1119-11e9-9f61-8f7c58c1afde.png)

With "layout-footer"
![image](https://user-images.githubusercontent.com/1127397/50730344-3aced980-1119-11e9-9792-5e3093fcf3d4.png)